docs/trace_collection.md

```diff
@@ -271,6 +271,67 @@ val request = Request.Builder()
 
 **Note**: If you use multiple Interceptors, this one must be called first.
 
+### RxJava
+
+To provide a continuous trace inside a RxJava stream you need to follow the steps below:
+1. Add the [OpenTracing for RxJava][8] dependency into your project and follow the **Readme** file 
+   for instructions. For example for a continuous trace you just have to add:
+   ```kotlin
+   TracingRxJava3Utils.enableTracing(GlobalTracer.get())
+   ```
+2. Then in your project open a scope when the Observable is subscribed and close it when it completes. Any span
+   created inside the stream operators will be displayed inside this scope (parent Span):
+   ```kotlin
+    Single.fromSupplier{}
+        .subscribeOn(Schedulers.io())
+        .map { 
+            val span = GlobalTracer.get().buildSpan("<YOUR_OP_NAME>").start()
+            // ...
+            span.finish()
+        }
+       .doOnSubscribe {
+           val span = GlobalTracer.get()
+               .buildSpan("<YOUR_OP_NAME>")
+               .start()
+           GlobalTracer.get().scopeManager().activate(span)
+       }
+       .doOnTerminate {
+           GlobalTracer.get().scopeManager().activeSpan()?.let {
+               it.finish()
+           }
+       }
+   ```
+### RxJava + Retrofit
+For a continuous trace inside a RxJava stream that uses Retrofit for the network requests:
+1. Configure the [Datadog Interceptor](#okhttp)
+2. Use the [Retrofit RxJava][9] adapters to use synchronous Observables for the network requests:
+```kotlin
+   Retrofit.Builder()
+   .baseUrl("<YOUR_URL>")
+   .addCallAdapterFactory(RxJava3CallAdapterFactory.createSynchronous())
```

Comment:
should we write about async `.create()` and its limitations?

Reply:
I think it is inferred automatically from the statement that they should use `createSynchronous`